src/utils/index.tsx

```diff
@@ -23,15 +23,15 @@ export const compareResource = (x: Resource, y: Resource) => {
   return 0;
 };
 
-/**
+/*
  * Compare user fullname by alphabetical order.
  */
-export const compareUser = (x: User, y: User) => {
-  const xName = fullName(x);
-  const yName = fullName(y);
-  if (xName > yName) {
+export const compareUser = (user1: User, user2: User) => {
```

Comment:
Can we delegate the logic to a generic string comparison, since `compareUser` and `compareSenior` both compare based on string?